src/pages/home/style.css

```diff
@@ -0,0 +1,30 @@
+.landing-page {
+    margin-top: 21rem;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    font-size: 48px;
+}
+
+.signup-btn {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    margin-top: 4rem;
+
+}
+
+.signup-btn button {
+    background-color: #AFD91A;
+    border-radius: 32px;
+    font-size: 24px;
+    padding: 1rem;
+    font-family: 'Gilda Display', serif;
+}
```

Comment:
Of course, if you agree with my comment about the sign-up button - we can re-use the `StyledButton` and you can customize any styles needed for that specific use using props and ultimately remove the stylings in this file for the signup button